### PR TITLE
feat(core): usable source event sent in callback

### PR DESCRIPTION
**Pull request by eregnier**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue-sse",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (tserkov) [inline package.json]: Bump this to `1.1.0` since this is a new functionality, not a bugfix.

File: README.md [hunk 2]
```diff
@@ -165,12 +165,12 @@ export default {
         });
 
         // Listen for messages without a specified event
-        msgServer.subscribe('', data => {
+        msgServer.subscribe('', (data, messageEvent) => {
```
Comment (tserkov) [inline README.md]: Rename `messageEvent` to `rawEvent` for clarity.

File: README.md [hunk 3]
```diff
@@ -165,12 +165,12 @@ export default {
...
           console.warn('Received a message w/o an event!', data);
         });
 
         // Listen for messages based on their event (in this case, "chat")
-        msgServer.subscribe('chat', message => {
+        msgServer.subscribe('chat', (message, messageEvent) => {
```
Comment (tserkov) [inline README.md]: Rename `messageEvent` to `rawEvent` for clarity.

**Comment by eregnier** [comment]
I made changes to my PR accordingly. Glad to contribute ;)

**Pull request merged**